src/codegen/compile_classical.ml

```diff
@@ -10937,9 +10936,8 @@ and compile_prim_invocation (env : E.t) ae p es at =
 
          StackRep.of_arity return_arity,
          code1 ^^ StackRep.adjust env fun_sr SR.Vanilla ^^
+         Closure.prepare_closure_call env ^^ (* FIXME: move to front elsewhere too *)
```

Comment:
So this now storing the prepared (forwarded closure in clos and re_loading it later to pass to call_closure (already forwarded). I guess that's ok. @luc-blaeser ?